source/macos/AdaptiveCards/AdaptiveCardsTests/Views/ACRTextFieldTests.swift

```diff
@@ -23,7 +22,7 @@ class ACRTextFieldTests: XCTestCase {
     }
     
     func testNoClearButtonDefaultRenderConfig() {
-        textField = ACRTextField(frame: .zero, config: RenderConfig.default)
+        textField = ACRTextField(textFieldConfig: InputFieldConfig.default)
         XCTAssertTrue(textField.subviews.isEmpty)
```

Comment:
```suggestion
        textField = ACRTextField(textFieldConfig: .default)
        XCTAssertTrue(textField.subviews.isEmpty)
```
Nit: Class name can be omitted here.